### DISCUSSION TITLE
Screengrab BuildFailure on openSUSE

### DISCUSSION THREAD

**Discussion by sfalken** [lxqt/lxqt#2291] [Q&A]
I've been out of the packaging game for a few years, but I'm getting back into it, with lxqt on openSUSE.   At the moment, I can't get screengrab to build.    If it were five years ago, I could probably look at this buildlog and figure out why its not building (I suspect it's a missing library in the build environment), but sadly, I'm out of practice and can't seem to sort out what the issue is.

Can somebody have a look, and tell me if they have some idea why this is failing to build?

https://susepaste.org/76301050

**Comment by tsujan** [comment]
Oh, this is https://github.com/lxqt/screengrab/issues/310, and you found the reason there.

**Comment by sfalken** [comment]
Yup.
